dev.zed.Zed.yaml

```diff
@@ -0,0 +1,105 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
+  - --device=dri
+
+  # Allow access to the user's home directory for reading and writing files
+  - --filesystem=home
+
+  # Allow shared memory access for inter-process communication
+  - --share=ipc
+
+  # Enable network access
+  - --share=network
+
+  # Provide access to X11 display server as a fallback
+  - --socket=fallback-x11
+
+  # Enable access to PulseAudio sound server
+  - --socket=pulseaudio
+
+  # Enable access to Wayland display server
+  - --socket=wayland
+
+  # Enable access to Canonical AppMenu
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+
+  # Enable access to D-Bus session bus (--socket=session-bus is not supported by Flathub's build system)
+  - --system-talk-name=org.freedesktop.DBus
+  - --talk-name=org.freedesktop.DBus
+
+  # Enable access to Flatpak host commands (not supported by Flathub's build system)
+  - --talk-name=org.freedesktop.Flatpak
+
+  # Enable access to Freedesktop Notifications
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
This is not needed anymore.

Zed uses Git heavily; you are going to need `--socket=ssh-auth` and `--talk-name=org.gnome.keyring.SystemPrompter`, along with `openssh`, `shared-modules/libsecret/libsecret.json`, and `gcr`.

Reply:
`--socket=gpg-agent` too

Reply:
Extra `finish-args` were added, but I'm unsure what you mean regarding `openssh` if we are already using `flatpak-spawn` access and `Sdk` as the runtime?

Reply:
Dependencies added!